dependencies.props

```diff
@@ -158,7 +158,7 @@
     </StaticDependency>
 
     <StaticDependency Include="Microsoft.TargetingPack.Private.WinRT">
-      <Version>1.0.3</Version>
+      <Version>1.0.5</Version>
```

Comment:
I replied on the original issue that I would follow up separately. It took long enough to get this change to work down level that I didn't want to add anything that wasn't strictly necessary to the change.